product.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>THEGYM | BEM Part 3</title>
+    <link rel="stylesheet" href="./assets/style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+</head>
+<body>
+    <nav class="nav">
+        <img src="./assets/img/logo.svg" alt="" class="nav__logo">
+        <div class="nav__menus">
+            <a class="nav__menu" href="/">Home</a>
+            <a class="nav__menu">Product</a>
+            <a class="nav__menu" href="./pricing.html">Pricing</a>
+            <div class="nav__menu">About Us</div>
+            <div class="nav__menu">Blog</div>
+            <div class="nav__menu">Contact</div>
+            <button class="nav__trial-btn">Free trial</button>
+        </div>
+    </nav>
+
+    <section class="prod-hero">
+        <div class="prod-hero__contents">
+            <div class="prod-hero__title">Grow your Sales and Services</div>
+            <div class="prod-hero__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat.</div>
+            <div class="prod-hero__btns">
+                <button class="prod-hero__start-btn">Get Started</button>
+                <button class="prod-hero__contact-btn">Contact Us</button>
+            </div>
+        </div>
+        <div class="prod-hero__img-container">
+            <img src="./assets/img/dashb.png" alt="" class="prod-hero__img">
+        </div>
+        <img src="./assets/img/product/hero-gradient.svg" alt="" class="prod-hero__gradient">
+    </section>
+
+    <section class="p-20 flex justify-between">
+        <img src="./assets/img/partner1.svg" alt="">
+        <img src="./assets/img/partner2.svg" alt="">
+        <img src="./assets/img/partner3.svg" alt="">
+        <img src="./assets/img/partner4.svg" alt="">
+        <img src="./assets/img/partner5.svg" alt="">
+    </section>
+
+    <section class="prod-features">
```

Comment:
Same naming issue here too.  Adjust `prod-..` name to be more descriptive for this section.